app/lib/actions/katello/content_view_version/export.rb

```diff
@@ -20,50 +20,21 @@ def plan(content_view_version:, destination_server: nil,
                  fail_on_missing_content: false)
+
           action_subject(content_view_version)
 
           sequence do
-            smart_proxy = SmartProxy.pulp_primary!
-            from_content_view_version = from_history&.content_view_version
-            export_service = ::Katello::Pulp3::ContentViewVersion::Export.new(
-                                                   smart_proxy: smart_proxy,
-                                                   content_view_version: content_view_version,
-                                                   destination_server: destination_server,
-                                                   from_content_view_version: from_content_view_version)
-            export_service.validate!(fail_on_missing_content: fail_on_missing_content,
-                                     validate_incremental: validate_incremental)
-
-            action_output = plan_action(::Actions::Pulp3::ContentViewVersion::CreateExporter,
-                                   content_view_version_id: content_view_version.id,
-                                   smart_proxy_id: smart_proxy.id,
-                                   destination_server: destination_server).output
-
-            plan_action(::Actions::Pulp3::ContentViewVersion::Export,
-                                   content_view_version_id: content_view_version.id,
-                                   smart_proxy_id: smart_proxy.id,
-                                   exporter_data: action_output[:exporter_data],
-                                   chunk_size: chunk_size,
-                                   from_content_view_version_id: from_content_view_version&.id)
-
-            history_output = plan_action(
-              ::Actions::Pulp3::ContentViewVersion::CreateExportHistory,
-              smart_proxy_id: smart_proxy.id,
-              exporter_data: action_output[:exporter_data],
-              pulp_href: action_output[:exporter_data][:pulp_href],
-              content_view_version_id: content_view_version.id,
-              from_content_view_version_id: from_content_view_version&.id,
-              destination_server: destination_server
-            ).output
-            plan_action(::Actions::Pulp3::ContentViewVersion::DestroyExporter,
-              smart_proxy_id: smart_proxy.id,
-              exporter_data: action_output[:exporter_data])
-            plan_self(exporter_data: action_output[:exporter_data], smart_proxy_id: smart_proxy.id,
-              destination_server: destination_server,
-              content_view_version_id: content_view_version.id,
-              from_content_view_version_id: from_content_view_version&.id,
-              export_history_id: history_output[:export_history_id],
-              exported_file_checksum: history_output[:exported_file_checksum],
-              path: history_output[:path])
+            export_output = plan_action(::Actions::Pulp3::Orchestration::ContentViewVersion::Export,
```

Comment:
wonder if you can just capture all the plan params to an options hash and pass that to plan action.
That way it becomes amply clear that this is like a delegating action